/**:
  ros__parameters:
    straight_p_gain: 0.8
    straight_i_gain: 0.002
    straight_lugre_fc: 0.02
    straight_rtz_offset: 0.085
    curve_p_gain: 0.4
    curve_i_gain: 0.002
    curve_lugre_fc: 0.01
    curve_rtz_offset: 0.085
    min_steer_thresh: 0.1
    max_steer_thresh: 0.4
    p_gain_rate_max: 1.5
    p_gain_rate_min: -5.0
    i_gain_rate_max: 0.05
    i_gain_rate_min: -0.05
    lugre_fc_rate_max: 0.1
    lugre_fc_rate_min: -0.1
    rtz_offset_rate_max: 0.5
    rtz_offset_rate_min: -0.5
